com.seafile.Client.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: com.seafile.Client
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: seafile-applet
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --persist=.ccnet
```

Comment:
Qt complains about not being able to get GPU acceleration, adding `--device=dri` fixes that.

Reply:
Fixed